pkg/crc/version/version.go

```diff
@@ -23,6 +23,8 @@ var (
 
 	// OC version which should be used.
 	ocVersion = "0.0.0-unset"
+
+	defaultOcURLBase = "0.0.0-unset"
```

Comment:
This doesn't feel like belongs here, but am OK

Reply:
True, it was a convenient place to put it.  Since it was no longer a constant, it needed to be somewhere with external configuration.